src/main/java/selenium/Selenium.java

```diff
@@ -0,0 +1,88 @@
+package selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Selenium {
+    static final String username = "TestUser01";
```

Comment:
naming convention says that constants names should be all caps with underscore between words 
private static final String USER_NAME

Reply:
fixed